CONTRIBUTING.md

```diff
@@ -125,6 +125,59 @@ When reviewing a PR, a maintainer will:
    they have acknowledged the message and will make the change.
  - Approve a PR which is ready to merge, or nearly ready with some minor tweaks or optional improvements.
 
+### Sign off
```

Comment:
N.B. just to mention that we link to the Synapse sign off page above. You might wanna adjust that link. https://github.com/matrix-org/matrix-appservice-bridge/pull/502/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R94

Reply:
Oh yes good catch, this is as good a time as any to update that link. Thank you!